.travis.yml

```diff
@@ -2,17 +2,22 @@ language: python
 python:
 - 2.7
 - 3.6
-- 3.7
+- "nightly"
 matrix:
   allow_failures:
-    - python: 3.7
+    - python: "nightly"
 
 sudo: false
 
 env:
   global:
     - MASTER_BRANCH=master
     - PYPI_PY=3.6  # deploy to pypi from python 3.6
+  matrix:
```

Comment:
I split up the tests here (it is helpful for diagnosing failures to have them be tested separately)